.github/workflows/production_deploy.yml

```
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - prod
+
+jobs:
+  move_to_production:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)